examples/hello-triangle-kdgui/app.cpp

```diff
@@ -200,7 +199,7 @@ void Test::App::OnResize(uint32_t width, uint32_t height)
     desc.SetVS(vs);
     desc.SetPS(ps);
     desc.SetRenderPass(render_pass);
-    pipeline = device.CreateGraphicsPipeline(std::move(desc), ia);
+    pipeline = device.CreateGraphicsPipeline(desc, ia);
```

Comment:
why copy?

Reply:
I changed CreateGraphicsPipeline to take the desc as a const reference, so theres no need to move.